### ISSUE TITLE
inittab does not support command chaining with &&, || or ;

### ISSUE THREAD

**Issue by M95D** [slicer69/sysvinit#25]
Hi.

Quote from the inittab man page:
> Please note that including certain characters in the process field will result in init attempting to launch a shell to interpret the command contained in the process field.  The characters which will trigger a shell are: ˜‘!$ˆ&*()=|}[];

According to this quote, inittab should support command chaining.
Add this to /etc/inittab:
```
ww:c:once:echo TEST1a ; echo TEST1b
wx:c:once:echo TEST2a && echo TEST2b
wy:c:once:/bin/false || echo TEST3
wz:c:once:echo TEST4 > /dev/tty11
```
Then run: `telinit c`
Expected output:
```
TEST1a
TEST1b
TEST2a
TEST2b
TEST3
```
... on current console and:
```
TEST4
```
... on tty11

Actual output:
```
TEST1a
TEST2a
```
There is no TEST1b, TEST2b, TEST3 or TEST4.
Note that character `>` isn't documented to trigger a shell, but I belive it should.

I found this bug on Gentoo Linux with OpenRC init. [Gentoo bug report](https://bugs.gentoo.org/939684).
This bug report appears to be related: https://github.com/slicer69/sysvinit/issues/1

Thank you.

**Comment by slicer69** [comment]
This is interesting, could you tell me which version of init you are running? 

When I run the above test, I get a different result. I get no output at all on the current terminal. Presumably because the shell performing the echo is running in the background. But I do get TEST4 appearing on tty11 as expected because it's expecitly redirected.

**Comment by slicer69** [comment]
Update: The characters "<" and ">" which will trigger a shell when used in an inittab file were written in the manual page. However, they were (ironically) not properly escaped, so the "man" command wasn't displaying them.

I've fixed the manual page so the escape characters are no longer hidden. This change is in the 3.11 branch.

**Comment by slicer69** [comment]
Additional update: I looked into this a bit more, writing the test a little to redirect everything to a TTY.

ww:c:once:echo TEST1a ; echo TEST1b > /dev/tty6
wx:c:once:echo TEST2a && echo TEST2b > /dev/tty6
wy:c:once:/usr/bin/false || echo TEST3 > /dev/tty6
wz:c:once:echo TEST4 > /dev/tty6

With this test, I was getting _only_ "TEST4" printed to the console. This confirms command chaining was failing, but single commands with redirect worked.

I then made a small change to init.c, removing the "exec" prefix on commands. Now, with the above test lines in inittab, I'm getting the following output to the console:

TEST1b
TEST2b
TEST4
TEST3

This matches expectations. The redirected output from the first two lines are bring printed. The "false or TEST3" test results in a printed line, and the single-command TEST4 is printed as well.

TEST1a and TEST2a are not printed to the console because the shell is executed in the background and has no console to print to, and isn't explicitly redirected.

Anyway, I think this closes both issues - both the manual page typo and the chaining of commands. I'll update the changelog and push out this fix to the 3.11 branch.
